backend/persistence/migrations/20241118114500_change_webauthn_credentials.up.fizz

```diff
@@ -0,0 +1,16 @@
+create_table("webauthn_credential_user_handles") {
+    t.Column("id", "uuid", {primary: true})
+    t.Column("user_id", "uuid", {"null": false})
+    t.Column("handle", "string", {"null": false, "unique": true})
+    t.Timestamps()
+    t.Index(["id", "user_id"], {"unique": true})
+    t.ForeignKey("user_id", {"users": ["id"]}, {"on_delete": "cascade", "on_update": "cascade"})
+}
+
+add_column("webauthn_credentials", "user_handle_id", "uuid", { "null": true })
+add_foreign_key("webauthn_credentials", "user_handle_id", {"webauthn_credential_user_handles": ["id"]}, {
+    "on_delete": "cascade",
```

Comment:
I would tend to set the `user_handle_id` to `NULL` when the referenced entry in `webauthn_credential_user_handles` is deleted, as assigning a user handle is optional. Conversely, I think it makes sense to drop the user handle when the credential itself is deleted.
```suggestion
    "on_delete": "set null",
```